src/Service/Validator.php

```diff
@@ -27,31 +27,36 @@ class Validator
             '>=2.2.5 <2.2.10' => '>=7.0 <7.2', //'~7.0.13|~7.1.0',
             '>=2.2.10 <2.3.0' => '>=7.1 <7.3', //'~7.1.0|~7.2.0',
             '>=2.3.0 <2.3.3' => '>=7.1 <7.3',   //'~7.1.3 || ~7.2.0',
-            '>=2.3.3' => '>=7.1 <7.4',       //  '~7.1.3||~7.2.0||~7.3.0'
+            '>=2.3.3 <2.4.0' => '>=7.1 <7.4',   //  '~7.1.3||~7.2.0||~7.3.0'
+            '>=2.4.0' => '>=7.3 <7.5', // '~7.3.0||~7.4.0'
         ],
         ServiceInterface::NAME_DB => [
-            '*' => '>=10.0 <10.3',
+            '<2.4.0' => '>=10.0 <10.3',
+            '>=2.4.0' => '10.4',
         ],
         ServiceInterface::NAME_NGINX => [
             '*' => '^1.9.0',
         ],
         ServiceInterface::NAME_VARNISH => [
             '<2.2.0' => '~3.5.0 || ^4.0',
             '>=2.2.0 <2.3.3' => '^4.0 || ^5.0',
-            '>=2.3.3' => '^4.0 || ^5.0 || ^6.2',
+            '>=2.3.3 <2.4.0' => '^4.0 || ^5.0 || ^6.2',
+            '>=2.4.0' => '^6.2',
```

Comment:
Same comment here as on DB. New version was added but are we sure old were dropped?